extras/cloudsql/setup_workload_identity.sh

```diff
@@ -1,3 +1,5 @@
+# !/bin/bash
+# [START gke_cloudsql_setup_workload_identity]
```

Comment:
Do snippets start before or after the copyright notice, usually? (this is not necessary the only file with this comment, just the first one I saw)

Reply:
good catch, Ill move them down! 👍